Added palm-rest files

arian153 opened the pull request:
stl, dwg, sldprt files

bstiq commented:
Looks good, I moved the files and added details in the mods/readme file. Also took the opportunity to clean that file up a bit.

Would you maybe have a picture of the rests in action ?

arian153 commented:
Okay, I will upload picture on discord tonight.Sent from my iPhoneOn Dec 5, 2023, at 01:42, ***@***.*** wrote:﻿
Looks good, I moved the files and added details in the mods/readme file. Also took the opportunity to clean that file up a bit.
Would you maybe have a picture of the rests in action ?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

arian153 commented:
Hi, I updated readme and photo!
